datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -96,7 +104,8 @@ object MVHelper {
           children = None,
           rawSchema = rawSchema)
       }
-    }
+    }.distinct
```

Comment:
Please recheck whether after applying distinct order of columns changing, we need to maintain the order of columns

Reply:
yes, thanks for the input, i cross verified this, it remains same order, as distinct creates new set which contains the first occurance of every element of the input sequence. So if its a,b,b,c,c,d,e, after distinct it remains same order as a,b,c,d,e